plugins/browser-plugin-media/src/schemata.ts

```diff
@@ -0,0 +1,26 @@
+import { MediaPlayerEventType } from './types';
+
+export const MEDIA_PLAYER_SCHEMA_PREFIX = 'iglu:com.snowplowanalytics.snowplow/media_player';
+export const MEDIA_PLAYER_SCHEMA_SUFFIX = '/jsonschema/1-0-0';
+export const MEDIA_PLAYER_SCHEMA = MEDIA_PLAYER_SCHEMA_PREFIX + MEDIA_PLAYER_SCHEMA_SUFFIX;
+export const MEDIA_PLAYER_SESSION_SCHEMA = MEDIA_PLAYER_SCHEMA_PREFIX + '_session' + MEDIA_PLAYER_SCHEMA_SUFFIX;
+export const MEDIA_PLAYER_AD_SCHEMA = MEDIA_PLAYER_SCHEMA_PREFIX + '_ad' + MEDIA_PLAYER_SCHEMA_SUFFIX;
+export const MEDIA_PLAYER_AD_BREAK_SCHEMA = MEDIA_PLAYER_SCHEMA_PREFIX + '_ad_break' + MEDIA_PLAYER_SCHEMA_SUFFIX;
+
+function eventNameForEventType(eventType: MediaPlayerEventType): string {
+  switch (eventType) {
+    case MediaPlayerEventType.AdFirstQuartile:
+      return 'ad_quartile';
+    case MediaPlayerEventType.AdMidpoint:
+      return 'ad_quartile';
+    case MediaPlayerEventType.AdThirdQuartile:
+      return 'ad_quartile';
+  }
+
+  return eventType;
+}
+
+export function getMediaPlayerEventSchema(eventType: MediaPlayerEventType) {
```

Comment:
```suggestion
export function getMediaPlayerEventSchema(eventType: MediaPlayerEventType): string {
```
bit of a nitpick, but it's nice to see at a glance at the signature what/if type a function is expected to return